### ISSUE TITLE
Container does not restart properly after running "docker stop"

### ISSUE THREAD

**Issue by kcraft** [CiscoCloud/haproxy-consul#25]
Runnung `docker stop haproxy-consul` followed by `docker start haproxy-consul` results in the following error in the logs:

```
ln: /consul-template/template.d/haproxy.tmpl: File exists
```

It looks like this line in launch.sh needs to remove the link before creating it or needs to test for the existence of the link:

```
ln -s /consul-template/template.d/${HAPROXY_MODE}.tmpl /consul-template/template.d/haproxy.tmpl
```

The same problem happens if the docker daemon or the VM it runs on are restarted.

**Comment by arhea** [comment]
+1 we have encountered this as well.

**Comment by chazsconi** [comment]
I believe this is fixed by my PR https://github.com/CiscoCloud/haproxy-consul/pull/27

**Comment by BrianHicks** [comment]
Should be fixed by #27. Thanks @chazsconi!
